docs/supported-sources/athena.md

```diff
@@ -0,0 +1,38 @@
+# Athena
+[Athena](https://aws.amazon.com/athena/) is an interactive query service that allows users to analyze data directly in Amazon S3 using standard SQL.
+
+The Athena destination stores data as Parquet files in S3 buckets and creates external tables in AWS Athena.
+
+ingestr supports Athena as a destination.
+
+## URI format
+The URI format for Athena is as follows:
+
+```plaintext
+athena://?bucket=<your-destination-bucket> \
+    query_result_path=<your-query-results-location> \
```

Comment:
`query_results_path` instead of `query_result_path`, that's what we use in Bruin CLI.

Reply:
ok